lib/experimental/Layouts/Utils/helper.ts

```diff
@@ -0,0 +1,29 @@
+const avatarColors: Record<string, string> = {
```

Comment:
We're widening the types here, and this is not a good idea, as it will allow any color. Instead, use the approach mentioned before:

```tsx
type AvailableColors = keyof typeof avatarColors
```

Reply:
You could maybe do something like:

```suggestion
const avatarColors: Record<AvailableColors, string> = {
```

Reply:
That's a circular dependency

Reply:
Remember that AvailableColors = keyof typeof avatarColors

Reply:
Yeah, that creates an error somewhere else, but let me see if I can find another solution to that

Reply:
Keep in mind I'm trying to keep the same function we have on the Frontend to keep some level of consistency. Just updated a new version of the types, check it out please. Thank you.

Reply:
I understand, but this whole project is about rethinking the foundations. It might make sense to keep a similar signature, but let's not get carried away or we can easily replicate some of the mistakes as well :)